pallets/phala/src/stakepool.rs

```diff
@@ -541,6 +543,35 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// manually assign rewards to pools by root for fixing issue 763
+		///
+		///Reuires:
+		/// 1. The caller is root
+		/// 2. assigned pool must currently exist
+		/// 3. reeward is positive
+		#[pallet::weight(0)]
+		pub fn force_assign_reward(
+			origin: OriginFor<T>,
+			reward_arr: Vec<(u64, BalanceOf<T>)>,
+		) -> DispatchResult {
+			// origin must be root
+			T::MiningSwitchOrigin::ensure_origin(origin)?;
+			for pair in reward_arr {
+			    let (pid, reward) = pair;
+			    //assigned pool must exist
```

Comment:
```suggestion
			    // The assigned pool must exist
```

Space after `//`; upper case

Reply:
Please fix